homeassistant/components/websocket_api/http.py

```diff
@@ -89,38 +90,48 @@ def description(self) -> str:
     async def _writer(self) -> None:
         """Write outgoing messages."""
         # Exceptions if Socket disconnected or cancelled by connection handler
-        to_write = self._to_write
+        message_queue = self._message_queue
         logger = self._logger
-        wsock = self.wsock
+        send_str = self.wsock.send_str
+        loop = self.hass.loop
+        debug = logger.debug
```

Comment:
Please add a comment motivating why we need to store object attributes in variables.

Reply:
924cd6250c2c5f7f0e413d47fc1d174eb60a32db